crates/host/src/wasmbus/providers/mod.rs

```diff
@@ -395,6 +395,13 @@ async fn provider_command(path: &Path, host_data: Vec<u8>) -> anyhow::Result<pro
         let _ = child_cmd.env("RUST_LOG", rust_log);
     }
 
+    // Pass through any OTEL configuration options to the provider as well
+    for (k, v) in env::vars() {
+        if k.starts_with("OTEL_") {
```

Comment:
I think this should be safe to do, but wanted to call this out just to make sure we're ok with doing this

Reply:
Sounds good to me 👍🏻 

No more setting `OTEL_MY_GITHUB_PASSWORD` 😡